app/assets/javascripts/nrdb_cards.js.coffee.erb

```diff
@@ -47,6 +47,7 @@ $(document).on 'turbolinks:load', ->
 
     getPrintings = (request) =>
       request.data['fields[printings]'] = 'card_id,card_type_id,title,side_id,faction_id,minimum_deck_size,influence_limit,influence_cost'
+      1+1
```

Comment:
Was this committed accidentally?

Reply:
yep! i put that in to force an asset rebuild.  Reverted.